sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1427,30 +1427,6 @@ class DataFrame private[sql](
   def transform[U](t: DataFrame => DataFrame): DataFrame = t(this)
 
   /**
-   * Returns a new RDD by applying a function to all rows of this DataFrame.
-   * @group rdd
-   * @since 1.3.0
-   */
-  def map[R: ClassTag](f: Row => R): RDD[R] = rdd.map(f)
```

Comment:
Good question... Checked the Jenkins MiMA section of the build log of this PR, didn't see any lines related to DataFrame.